README.md

```diff
@@ -7,4 +7,35 @@ Based on https://strongloop.com/
 Requirements
 ------------
 
-Need a valid node.js setup. Either Stouts.nodejs or role-wcl-nodejs
\ No newline at end of file
+Need a valid node.js setup. Either Stouts.nodejs or role-wcl-nodejs
+
+To use:
+
+variables:
+
+* `strongloop_install`: if setting up strongloop and strong-pm
+* `strongloop_instances`: a list of config for each strongloop instance
+
+Note:
+
+For each config, `update` means if deploy this instance at this run.
+
+```
+---
+- hosts: app
+  vars_prompt:
+    - name: prompt_update_1
+      prompt: "Deploy this repo? (yes/no) default: yes"
```

Comment:
Don't include the `(yes/no) default: yes` - the string is auto-generated by ansible.